models/SentryAppender.cfc

```diff
@@ -51,7 +54,10 @@ component extends="coldbox.system.logging.AbstractAppender" accessors=true{
 				logger = loggerCat
 			);
 				
-		} else if( isStruct( extraInfo ) && extraInfo.keyExists( "exception" ) && isStruct( extraInfo.exception ) && extraInfo.exception.keyExists( "StackTrace" ) ){
+		} else if( 
+			( isStruct( extraInfo ) || isObject( extraInfo ) )
```

Comment:
@jclausen You removed the `&& extraInfo.keyExists( "exception" )`  check which is causing the build to fail.  Can you add that back in?

Reply:
Done